public/docs/dart/latest/guide/displaying-data.jade

```diff
@@ -431,16 +430,14 @@
 
     p.
       The app's entry point—<code>main.dart</code>—imports
-      <code>bootstrap.dart</code>.
+      <code>angular2.dart</code> and <code>bootstrap.dart</code>.
       Both <code>show_properties.dart</code> and <code>friends_service.dart</code>
-      import <code>angular2.dart</code> instead,
-      because they use Angular APIs but aren't entry points.
-      (They don't call <code>bootstrap()</code>.)
-      See <a href="/docs/dart/latest/quickstart.html#performance">Performance,
-      the transformer, and Angular 2 libraries</a> for more information.
+      import only <code>angular2.dart</code>
+      because they aren't entry points.
```

Comment:
I think we no longer need to talk about non-entry points. You no longer import `bootstrap.dart` simply because you wouldn't be using the `bootstrap` function. Dart analyzer will flag any code that imports `bootstrap.dart` and does not use it.

I think we can rephrase this as (roughly): "To bootstrap your Angular 2 application in your entry point import `bootstrap.dart` and call `bootstrap`. To define a component or a directive, import `angular2.dart` and use `@Comonent` or `@Directive` respectively".